bin/mesos-executor

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
-exec java $HADOOP_OPTS $EXECUTOR_OPTS -cp lib/*.jar org.apache.mesos.hdfs.MesosExecutor
+if [ -z "$JAVA_HOME" ]; then
```

Comment:
Lets name this variable JAVA_CMD or JAVA_EXEC, I'm wary of a variable called JAVA